CMakeLists.txt

```diff
@@ -29,4 +29,49 @@ export_var(LEATHERMAN_LIBRARIES)
 if(LEATHERMAN_ENABLE_TESTING)
     enable_testing()
     add_subdirectory(tests)
+
+    add_custom_target(cppcheck COMMAND cppcheck --enable=warning,performance --error-exitcode=2 --quiet ${CPPCHECK_DIRS})
+
+    include(FindPythonInterp)
```

Comment:
This could be better as a re-usable cmake macro for applying cpplint with our configuration.